packages/react-ui-core/src/Schema/Schema.js

```diff
@@ -0,0 +1,110 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import themed from 'react-themed'
+
+const ApartmentComplex = ({ children, ...rest }) => (
+  <div itemType="https://schema.org/ApartmentComplex" itemScope {...rest}>
+    {children}
+  </div>
+)
+
+ApartmentComplex.propTypes = {
+  children: PropTypes.node,
+}
+
+const AggregateRating = ({ score, count, reviews }) => {
+  if (!count && !reviews) {
+    return null
+  }
+
+  return (
+    <div
+      itemScope
+      itemProp="aggregateRating"
+      itemType="https://schema.org/AggregateRating"
+    >
+      <meta itemProp="worstRating" content="1.0" />
+      <meta itemProp="bestRating" content="5.0" />
+      <meta itemProp="ratingValue" content={score} />
+      {!!count && <meta itemProp="ratingCount" content={count} />}
+      {!!reviews && <meta itemProp="reviewCount" content={reviews} />}
+    </div>
+  )
+}
+
+AggregateRating.propTypes = {
+  score: PropTypes.number,
+  count: PropTypes.number,
+  reviews: PropTypes.number,
+}
+
+const Address = ({ addressLine1, city, state, zip }) => (
+  <span itemType="https://schema.org/PostalAddress" itemProp="address" itemScope>
+    <meta itemProp="streetAddress" content={addressLine1} />
+    <meta itemProp="addressLocality" content={city} />
+    <meta itemProp="addressRegion" content={state} />
+    <meta itemProp="postalCode" content={zip} />
+  </span>
+)
+
+Address.propTypes = {
+  addressLine1: PropTypes.string,
+  city: PropTypes.string,
+  state: PropTypes.string,
+  zip: PropTypes.string,
+}
+
+const Geo = ({ latitude, longitude }) => {
+  if (!latitude || !longitude) {
+    return null
+  }
+
+  return (
+    <div itemProp="geo" itemScope itemType="https://schema.org/GeoCoordinates">
+      <meta itemProp="latitude" content={latitude} />
+      <meta itemProp="longitude" content={longitude} />
+    </div>
+  )
+}
+
+Geo.propTypes = {
+  latitude: PropTypes.number,
+  longitude: PropTypes.number,
+}
+
+@themed(['PropertyLink'], { pure: true })
+class NameAndUrl extends PureComponent {
+  render() {
+    const { url, theme, children } = this.props
+    return (
+      <div itemProp="name">
+        {url ?
+          <a
```

Comment:
Does this have to be anchor tag? Or can we append it to the div tag instead? Or vice versa?

Reply:
Confirmed; needs to be an anchor =\

Reply:
Can the `itemProp="name"` be merged into the anchor tag?

Reply:
We want to have the least amount of dom nodes as possible as it pertains to a listing.

Reply:
Ah, I understand the concern now. So the problem is the conflicting `itemProp`s that need to be on separate tags. I'm definitely open to suggestions as I'm not sure how to get around having the two nodes here.

Reply:
Can you just do:
```
if (url) {
  return <a itemProp="name" href.... >{c
}

return <div itemProp="name">{children}</div>
```

Reply:
I guess my question would if it has a url does it also need an `itemProp="name"`

Reply:
It needs both in this case. This is the ideal scenario: 
![screen shot 2018-05-14 at 3 27 16 pm](https://user-images.githubusercontent.com/4341758/40018813-60f3816c-578b-11e8-94a5-0c2708873355.png)